src/wallet/wallettool.cpp

```diff
@@ -112,7 +113,7 @@ bool ExecuteWalletToolFunc(const std::string& command, const std::string& name)
             WalletShowInfo(wallet_instance.get());
             wallet_instance->Close();
         }
-    } else if (command == "info" || command == "salvage") {
+    } else if (command == "info" || command == "salvage" || command == "dump") {
```

Comment:
nit: What's similar about `info`, `salvage` and `dump`? Can we make it a one-level switch instead?

Reply:
Removed this `else if`. I think this is a left over of when `info` and `salvage` shared wallet opening code.